client/src/store/reducer/createSurveyReducer.js

```diff
@@ -37,6 +38,22 @@ const mappedQuestions = (questions, payload, editOptions) => {
   );
 };
 
+const switchQuestions = (questions, firstIndex, secondIndex) => {
```

Comment:
switchQuestions and switchOptions are identical functions, you've just given them different argument names!

Reply:
Also `elementToMove` is a bit misleading since two elements are really being changed.

Reply:
Also per our discussion with Michael this morning, avoid stating a variable's type in its name, e.g. "switchedQuestionList"

Reply:
Agreed, too much code repetition. Pass in the questions or options as an argument!